src/App.spec.js

```diff
@@ -0,0 +1,17 @@
+import { render, screen } from '@testing-library/react'
+import App from './App'
+
+it('renders the input from', () => {
+  render(<App />)
+  expect(screen.getByRole('textbox')).toBeInTheDocument()
+  screen.debug()
```

Comment:
I don't know anything about `.debug()` yet. Looking forward to learning from you! 😁

Reply:
i thought, it would display detailed error message for testing, but actually i'm not sure... 😅
The documentation tells sth different: https://testing-library.com/docs/queries/about/#screendebug

Reply:
i actually got this from Melissa and i thought it would give you a (more) detailed error message for failed tests... but actually i'm not quiet sure... didn't do that much testing jet... 🤭

Reply:
https://testing-library.com/docs/queries/about/#screendebug